solutions/devsprint-vinicius-carvalho-3/DeliveryApp/Screens/Components/MenuItemCell.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  MenuItemCell.swift
+//  DeliveryApp
+//
+//  Created by Manoel Filho on 21/07/22.
+//
+
+import UIKit
+
+final class MenuItemCell: UIView {
+    
+    var menuItem: MenuItem? {
+        didSet{
+            self.nameLabel.text = menuItem?.name
+            self.priceLabel.text = .currencyValue(value: menuItem?.price ?? 0)
+            self.productImage.image = UIImage(named: "pizza")
+        }
+    }
+    
+    private lazy var wrapperItemStackView: UIStackView = {
+        let wrapperItemStackView = UIStackView()
+        wrapperItemStackView.alignment = .center
+        wrapperItemStackView.translatesAutoresizingMaskIntoConstraints = false
+        return wrapperItemStackView
+    }()
+    
+    private lazy var descriptionItemStackView: UIStackView = {
+        let descriptionItemStackView = UIStackView()
+        descriptionItemStackView.axis = .vertical
+        descriptionItemStackView.spacing = 5
+        descriptionItemStackView.translatesAutoresizingMaskIntoConstraints = false
+        return descriptionItemStackView
+    }()
+    
+    private lazy var nameLabel: UILabel = {
+        let nameLabel = UILabel()
+        nameLabel.text = "Title"
+        nameLabel.font = .systemFont(ofSize: 15, weight: .medium)
+        nameLabel.tintColor = .black
+        nameLabel.translatesAutoresizingMaskIntoConstraints = false
+        return nameLabel
+    }()
+    
+    
+    private lazy var priceLabel: UILabel = {
+        let priceLabel = UILabel()
+        priceLabel.text = "0"
+        priceLabel.font = .systemFont(ofSize: 13, weight: .regular)
+        priceLabel.tintColor = .gray
+        priceLabel.translatesAutoresizingMaskIntoConstraints = false
+        return priceLabel
+    }()
+    
+    private lazy var productImage: UIImageView = {
+        let productImage = UIImageView()
+        productImage.translatesAutoresizingMaskIntoConstraints = false
+        productImage.clipsToBounds = true
+        productImage.contentMode = .scaleAspectFill
+        productImage.layer.cornerRadius = 10
+        return productImage
+    }()
+    
+    convenience init() {
+        self.init(frame: .zero)
+        setup()
+    }
+    
+}
+
+
+extension MenuItemCell: ViewCode {
+    
+    func setupSubviews() {
+        
+        descriptionItemStackView.addArrangedSubview(nameLabel)
+        descriptionItemStackView.addArrangedSubview(priceLabel)
+        
+        wrapperItemStackView.addArrangedSubview(descriptionItemStackView)
```

Comment:
[Suggestion]
Move inside on your StackView.

Reply:
Done. Using _arrangedSubviews_ option in stack definition